boilerplate/package.json

```diff
@@ -110,7 +110,7 @@
     "prettier": "2.6.2",
     "query-string": "^7.0.1",
     "react-devtools-core": "4.24.7",
-    "react-dom": "^18.0.0",
+    "react-dom": "18.0.0",
```

Comment:
Pinned this version because `npm install` was throwing a peer dependency warning.

Previously:
`"react": "18.0.0"` 
`"react-dom": "^18.0.0"` -> `"react-dom":"18.0.2"`

`"react-dom":"18.0.2"` has a peer dependency of `"react": "18.0.2"`, but only 18.0.0 was getting installed. 

Now: 
`"react": "18.0.0"` 
`"react-dom": "18.0.0"`

Reply:
Shouldn't we upgrade React to 18.0.2?

Reply:
That would also likely work, this change was made from another PR

Reply:
https://github.com/infinitered/ignite/pull/2003/files#diff-594f75b3ec0c76210a39b8c7ce238eea438454a93d830bd968a282f89e30a8b0

Reply:
Updating React to `18.0.2` causes more peerDependency errors, which would involve updating `react-native`. I think sticking with `18.0.0` for this PR would be the more stable change

<img width="1102" alt="Screen Shot 2022-08-17 at 3 27 39 PM" src="https://user-images.githubusercontent.com/37849890/185254619-4f97050f-b327-47f9-8612-b1b7117cb542.png">